tests/yam/validate/validate_kdump.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# SUSE's openQA tests
+#
+# Summary: Verify Kdump service status, Kdump kernel parameters
+#          and crush dump after triggering a kernel crash.
+#
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'consoletest';
+use power_action_utils 'power_action';
+use Utils::Architectures qw(is_s390x);
+use utils 'systemctl';
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
+
+    systemctl("is-active kdump.service", ignore_failure => 1);
```

Comment:
does it really ignore failure? it should fail if the service is not as expected.

Reply:
changed.